bundles/persistence/org.openhab.persistence.mongodb/java/org/openhab/persistence/mongodb/internal/MongoDBPersistenceService.java

```diff
@@ -285,8 +285,6 @@ private void disconnectFromDatabase() {
                 state = OpenClosedType.valueOf(obj.getString(FIELD_VALUE));
             } else if (item instanceof RollershutterItem) {
                 state = new PercentType(obj.getInt(FIELD_VALUE));
-            } else if (item instanceof ColorItem) {
```

Comment:
Explain or revert this change.

Reply:
The check for `ColorItem` is redundant because it comes after the check for `DimmerItem`. Again, I can move the check for `ColorItem` earlier if that is a solution you prefer.